src/Rules/PHPUnit/MockMethodCallRule.php

```diff
@@ -0,0 +1,85 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\PHPUnit;
+
+use PhpParser\Node;
+use PHPStan\Analyser\Scope;
+use PHPStan\Type\Constant\ConstantStringType;
+use PHPStan\Type\Generic\GenericObjectType;
+use PHPStan\Type\IntersectionType;
+use PHPStan\Type\ObjectType;
+use PHPUnit\Framework\MockObject\InvocationMocker;
```

Comment:
This class does not exist and might be causing https://github.com/phpstan/phpstan-phpunit/issues/73

Reply:
Should be fixed in https://github.com/phpstan/phpstan-phpunit/pull/74

Tests was running on phpunit ^7.0. The class was existing.